.azure/pipelines/ci.yml

```diff
@@ -607,7 +607,7 @@ stages:
       - script: ./build.cmd -ci -nobl -noBuildRepoTasks -restore -noBuild -noBuildNative -projects src/Grpc/**/*.csproj
         displayName: Restore interop projects
       - script: ./build.cmd -ci -nobl -noBuildRepoTasks -noRestore -test -all -noBuildNative -projects eng\helix\helix.proj
-                /p:IsRequiredCheck=true /p:IsHelixJob=true /p:BuildInteropProjects=true /p:RunTemplateTests=true
+                /p:IsHelixDaily=true /p:IsRequiredCheck=true /p:IsHelixJob=true /p:BuildInteropProjects=true /p:RunTemplateTests=true
```

Comment:
revert before mergning

Reply:
Can we make a bot command to manually run the helix daily tests on a PR?

Reply:
Good idea. I'll add a DOI item to track.